src/witan/gateway/handler.clj

```diff
@@ -74,13 +62,16 @@
 
 (defmethod handle-message
   :command
-  [ch {:keys [command/id command/key command/version] :as command} {:keys [kafka]}]
+  [ch {:keys [command/id command/key command/version] :as command} {:keys [kafka connections]}]
   (let [receipt (java.util.UUID/randomUUID)
         now (iso-dt-now)]
-    (swap! receipts assoc receipt {:ch ch :at (t/now)})
+    (p/add-receipt! connections (partial dispatch-event! ch receipt) receipt)
```

Comment:
Your question is a bit vague. Do you mean literally this line? Or the whole `handle-message` fn?